### ISSUE TITLE
XML test suite?

### ISSUE THREAD

**Issue by bansp** [freedict/fd-dictionaries#64]
I wonder if it makes sense to have an XML test suite, with examples of what needs to be supported by each potential modification of the Freedict schema, and of what is not legal.

**Comment by humenda** [comment]
I suppose you are the only one in the project to answer this and to implement
it. Maybe you are aware :).

**Comment by bansp** [comment]
Actually, it's not that simple. I've only been an end-user of test-driven development and have only a rough idea of how it could be implemented in practice. I would probably first look at how the current TEI implements its test suite, as that is very impressive. But I don't imagine being able to put time into learning this and then implementing this for Freedict. I think my limitations in this regard can be easily and quickly outdone by a willing comp-sci student.
In other words, this could be material for something like a semester project. Maybe I should come up with a label for that in the ticketing system...
